backend/store/cache/cache.go

```diff
@@ -47,6 +47,16 @@ func buildCache(resources []corev2.Resource, synthesize bool) cache {
 
 type resourceSlice []Value
 
+func (s resourceSlice) Find(value Value) Value {
+	idx := sort.Search(len(s), func(i int) bool {
+		return !resourceLT(s[i], value)
+	})
+	if idx < len(s) && s[idx].Resource.GetObjectMeta().Name == value.Resource.GetObjectMeta().Name {
+		return s[idx]
+	}
+	return Value{}
```

Comment:
Should this be `return nil`?

Reply:
AFAIK you cannot use `nil` as type Value here!